packages/usdk/packages/upstreet-agent/packages/react-agents/components/core/chat.tsx

```diff
@@ -27,6 +28,37 @@ export const ChatActions = () => {
         //   await e.commit();
         // }}
       />
+      <Action
```

Comment:
Does this mean we will add `messageReaction` support to the frontend?

Reply:
Yes the action method requires handling on the interfaces. Ideally implementing the clients reaction flow if supported otherwise a log event to the interface indicating the response